tutorials/02-functions/02-what-is-this.md

```diff
@@ -166,9 +179,9 @@ addThreeNumbers.apply('oogieboogie', numbers);
 
 `call()` and `apply()` are great ways of explicitly setting `this` when we call a function. Sometimes, however, we want to set `this` when we define a function, not when we call it.
 
-`call()` and `apply()` invoke the function immediately. `bind()` is different — it returns a copy of the function that takes the sometimes slippery `this` and replaces it with a hard set value that is above the three rules we set above.
+`call()` and `apply()` invoke the function immediately. `bind()` is different — it returns a copy of the function that takes the sometimes slippery `this` and replaces it with a hard set value that we give it.
```

Comment:
Is this an okay solution for the second part of #29?

I tried to think of a way to rephrase "above the three rules we set above", but I couldn't find a good phrasing. It feels strange to say it's "above the three rules" in any fashion. I feel like––if we're going off of the three rules––what `bind()` really does is use the logic of Rule 3 ("`this` can be set explicitly") to trump over Rule 2 ("`this` is set implicitly when a function is called as a method"). But that just seems confusing to say. It seems weird to phrase the use of `bind()`.